content/concepts/multiplex/overview.md

```diff
@@ -9,24 +9,45 @@ aliases:
 
 ## Overview
 
-Stream Multiplexing (_stream muxing_) is a way of sending multiple streams of data over one
-communication link. It combines multiple signals into one unified signal so it can be transported
-'over the wires', then it is demulitiplexed (_demuxed_) so it can be output and used by separate
-applications. This is done to share a single TCP connection using unique port numbers to distinguish
-streams, between the multiple proceeses (such as kademlia and gossipsub) used by applications (such as IPFS)
-to make connection and transmission more efficient.
-
-With muxing, libp2p applications may have many separate streams of communication between peers, as well as
-have multiple concurrent streams open at the same time with a peer. Stream multiplexing allows us to initialize
-and use the same [transport](/concepts/transports/overview) connection across the lifetime of our interaction with a
-peer. With muxing, we also only need to deal with [NAT traversal](/concepts/nat/overview) once to be able to open as
-many streams as we need, since they will all share the same underlying transport connection. Applications can enable
-support for multiple multiplexers, which will allow you to fall back to a widely-supported multiplexer if a preferred
-choice is not supported by a remote peer.
+Network protocols often use bi-directional message streams built on top of a stream
+abstraction to transmit data between nodes. However, relying on a single message
```

Comment:
```suggestion
libp2p is built on top of a stream abstraction and uses a bi-directional message stream to send data between peers.
However, relying on a single message
```

Reply:
The idea is to introduce stream multiplexing "in general" and then relate it to libp2p

Reply:
Understood, feel free to ignore